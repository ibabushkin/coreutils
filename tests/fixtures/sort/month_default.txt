Jan Lorem ipsum dolor sit amet
Dec consectetur adipiscing elit
May sed do eiusmod tempor incididunt
Oct ut labore et dolore magna aliqua
N/A Ut enim ad minim veniam, quis
JUN nostrud exercitation ullamco
mar laboris nisi ut aliquip ex ea
