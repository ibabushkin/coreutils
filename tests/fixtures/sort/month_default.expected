N/A Ut enim ad minim veniam, quis
Jan Lorem ipsum dolor sit amet
mar laboris nisi ut aliquip ex ea
May sed do eiusmod tempor incididunt
JUN nostrud exercitation ullamco
Oct ut labore et dolore magna aliqua
Dec consectetur adipiscing elit
